<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SelectionDialog</class>
 <widget class="QDialog" name="SelectionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>-131</y>
        <width>606</width>
        <height>556</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="2" column="1" colspan="3">
        <widget class="HistoryComboBox" name="fileNameComboBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>Accepts regular expression</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="4">
        <widget class="QWidget" name="dirWidget" native="true">
         <layout class="QGridLayout" name="gridLayout_2">
          <property name="margin">
           <number>0</number>
          </property>
          <item row="1" column="0">
           <widget class="QLabel" name="label">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Browse directory:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QPushButton" name="dirPushButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Browse</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1" colspan="2">
           <widget class="QLineEdit" name="dirLineEdit"/>
          </item>
          <item row="3" column="0" colspan="2">
           <widget class="QCheckBox" name="dirCheckBox">
            <property name="text">
             <string>Browse subdirectories</string>
            </property>
           </widget>
          </item>
          <item row="3" column="2" colspan="2">
           <widget class="QCheckBox" name="selectImportedCheckBox">
            <property name="text">
             <string>Select imported files</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="3" column="3">
        <widget class="HistoryComboBox" name="imageSizeComboBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>Accepts regular expression</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>File size</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="HistoryComboBox" name="fileSizeComboBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>Accepts regular expression</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Image size [px]</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0" colspan="4">
        <widget class="QGroupBox" name="exifGroupBox">
         <property name="title">
          <string>Check Exif metadata</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Processing Software</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="HistoryComboBox" name="exifCameraManufacturerComboBox">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="toolTip">
             <string>Accepts regular expression</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>Camera manufacturer</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>Camera model</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="HistoryComboBox" name="exifCameraModelComboBox">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="toolTip">
             <string>Accepts regular expression</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" colspan="3">
           <widget class="HistoryComboBox" name="exifSoftwareComboBox">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="toolTip">
             <string>Accepts regular expression</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="4" column="0" colspan="4">
        <widget class="QGroupBox" name="anyGroupBox">
         <property name="title">
          <string>Check any metadata</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="2" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Author</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QDateTimeEdit" name="digitizedBeforeDateTimeEdit"/>
          </item>
          <item row="0" column="3">
           <widget class="QDateTimeEdit" name="createdBeforeDateTimeEdit"/>
          </item>
          <item row="0" column="4">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Digitized between</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>and</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QDateTimeEdit" name="digitizedAfterDateTimeEdit"/>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Created between</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>and</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QDateTimeEdit" name="createdAfterDateTimeEdit"/>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Copyright</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1" colspan="4">
           <widget class="HistoryComboBox" name="authorComboBox">
            <property name="toolTip">
             <string>Accepts regular expression</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1" colspan="4">
           <widget class="HistoryComboBox" name="copyrightComboBox">
            <property name="toolTip">
             <string>Accepts regular expression</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="fileNameLabel">
         <property name="text">
          <string>File name</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0" colspan="4">
        <widget class="QGroupBox" name="iptcGroupBox">
         <property name="title">
          <string>Check IPTC matadata</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="3" column="0">
           <widget class="QLabel" name="label_15">
            <property name="text">
             <string>Country</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1" colspan="3">
           <widget class="HistoryComboBox" name="iptcKeywordsComboBox">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="toolTip">
             <string>Accepts regular expression</string>
            </property>
           </widget>
          </item>
          <item row="3" column="3">
           <widget class="HistoryComboBox" name="iptcCityComboBox">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="toolTip">
             <string>Accepts regular expression</string>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QLabel" name="label_16">
            <property name="text">
             <string>City</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1" colspan="3">
           <widget class="HistoryComboBox" name="iptcDescriptionComboBox">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="toolTip">
             <string>Accepts regular expression</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="HistoryComboBox" name="iptcCountryNameComboBox">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="toolTip">
             <string>Accepts regular expression</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_13">
            <property name="text">
             <string>Object name</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="HistoryComboBox" name="iptcObjectNameComboBox">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="toolTip">
             <string>Accepts regular expression</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_14">
            <property name="text">
             <string>Keywords</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_18">
            <property name="text">
             <string>Description</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_17">
            <property name="text">
             <string>Edit status</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="HistoryComboBox" name="iptcEditStatusComboBox">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="toolTip">
             <string>Accepts regular expression</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="0" colspan="4">
        <widget class="QCheckBox" name="clearSelectionCheckBox">
         <property name="toolTip">
          <string>Unselect items selected before this action</string>
         </property>
         <property name="text">
          <string>Clear current selection</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>HistoryComboBox</class>
   <extends>QComboBox</extends>
   <header>historycombobox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>scrollArea</tabstop>
  <tabstop>dirLineEdit</tabstop>
  <tabstop>dirPushButton</tabstop>
  <tabstop>dirCheckBox</tabstop>
  <tabstop>selectImportedCheckBox</tabstop>
  <tabstop>fileNameComboBox</tabstop>
  <tabstop>fileSizeComboBox</tabstop>
  <tabstop>imageSizeComboBox</tabstop>
  <tabstop>anyGroupBox</tabstop>
  <tabstop>createdAfterDateTimeEdit</tabstop>
  <tabstop>createdBeforeDateTimeEdit</tabstop>
  <tabstop>digitizedAfterDateTimeEdit</tabstop>
  <tabstop>digitizedBeforeDateTimeEdit</tabstop>
  <tabstop>authorComboBox</tabstop>
  <tabstop>copyrightComboBox</tabstop>
  <tabstop>exifGroupBox</tabstop>
  <tabstop>exifSoftwareComboBox</tabstop>
  <tabstop>exifCameraManufacturerComboBox</tabstop>
  <tabstop>exifCameraModelComboBox</tabstop>
  <tabstop>iptcGroupBox</tabstop>
  <tabstop>iptcObjectNameComboBox</tabstop>
  <tabstop>iptcKeywordsComboBox</tabstop>
  <tabstop>iptcDescriptionComboBox</tabstop>
  <tabstop>iptcCountryNameComboBox</tabstop>
  <tabstop>iptcCityComboBox</tabstop>
  <tabstop>iptcEditStatusComboBox</tabstop>
  <tabstop>buttonBox</tabstop>
  <tabstop>clearSelectionCheckBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SelectionDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SelectionDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
